services/harmony/test/jobs/job-sharing.ts

```diff
@@ -11,6 +11,10 @@ const notJobOwner = 'jill'; // jill wants to access the results of joe's jobs
 
 const collectionWithEULAFalseAndGuestReadTrue = 'C1233800302-EEDTEST';
 const collectionWithEULATrueAndGuestsReadTrue = 'C1233860183-EEDTEST';
+// All collections in EEDTEST are in the All Collections permission group,
+// which provides read and order permissions to all guest users. This means
+// if a fixture is regenerated for C1233147317-EEDTEST it will need to be
+// manually updated to remove the "read" permission.
```

Comment:
I spent a little while on this one, too. It's not very satisfying to manually tweak the CMR response, but that permission that is set for all collections in EEDTEST will mean there's no EEDTEST collection we can use without guest read permissions.